tests/api/expected-outputs/augsburg_ar_events.json

```
@@ -315,6 +320,7 @@
       "recurrence_id": 1,
       "timezone": "Europe/Berlin"
     },
-    "hash": null
+    "hash": null,
+    "recurrence_rule": "DTSTART:20300101T120000\nRRULE:FREQ=WEEKLY;INTERVAL=2;UNTIL=20310101T235959;BYDAY=TU,TH"
   }
-]
+]
```

```suggestion
]

```
I'd prefer newlines at the end of files